src/content/datachannel/filetransfer/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<html>
+<head>
+
+  <meta name="keywords" content="WebRTC, HTML5, JavaScript">
+  <meta name="description" content="Client-side WebRTC code samples.">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta id="viewport" name="viewport" content="width=device-width, initial-scale=1">
+
+  <base target="_blank">
+
+  <title>RTCDataChannel</title>
+
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="../../../css/main.css">
+  <link rel="stylesheet" href="css/main.css" />
+
+</head>
+
+<body>
+
+  <div id="container">
+
+    <h1><a href="https://googlechrome.github.io/webrtc/" title="WebRTC samples homepage">WebRTC samples</a> <span>RTCDataChannel Filetransfer</span></h1>
+
+    <form id="fileInfo">
+      <input type="file" id="fileInput" name="files"/>
+    </form>
+
+    Send progress: <progress id="sendProgress" max="0" value="0"></progress>
```

Comment:
Could you add a little more space (margin: 0 0 1em 0) between each or these lines? If you can be bothered, maybe put the progress elements on the same line as the label text, like http://webrtc.github.io/samples/src/content/getusermedia/volume.